frappe_appointment/patches/v0_1/change_fieldtype_to_duration.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+
+def execute():
```

Comment:
This will not work. Also, what is the problem with the time field? Why do we want this migration?

And if we are going to use the same field, then it is not ideal for change, as it will break the system where the app is installed. The ideal way to do this is to create a new field with a different name and then copy the value and remove the old column after copying the value.

<img width="995" alt="image" src="https://github.com/user-attachments/assets/56ea2f15-52b6-484b-9416-0fc3ab6ede0c" />

Reply:
I'll check this out. When I tested, it successfully migrated. 

The reason to migrate to Duration field was that Time field is a little buggy, and doesn't make sense for a input that expects a duration.

Reply:
@zeel-codder Fixed this in latest commit. Can you please test again?

Reply:
`04:00:00.000000` was convert to `40000.000000000` 🤔 ?

Also, `reload_doc` is one more hack it just read the doc schema form the file so may break in case of there is an issue on file 🤔.  But as this is our DOCTYPE then it should work.

@shivamsn97

Reply:
Issue was in my local, working fine now